src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PlacementRequestService.kt

```diff
@@ -128,13 +129,14 @@ class PlacementRequestService(
         expectedArrival = it.expectedArrival,
         duration = it.duration,
       )
-      this.createPlacementRequest(placementRequirements, placementDates, notes)
+      val isParole = placementApplicationEntity.placementType == PlacementType.RELEASE_FOLLOWING_DECISION
+      this.createPlacementRequest(placementRequirements, placementDates, notes, isParole)
     }
 
     return AuthorisableActionResult.Success(placementRequests)
   }
 
-  fun createPlacementRequest(placementRequirements: PlacementRequirementsEntity, placementDates: PlacementDates, notes: String?): PlacementRequestEntity {
+  fun createPlacementRequest(placementRequirements: PlacementRequirementsEntity, placementDates: PlacementDates, notes: String?, isParole: Boolean?): PlacementRequestEntity {
```

Comment:
Does `isParole` actually need to be nullable?

Reply:
No, probably not now